src/test/java/eggventory/StockTypeTest.java

```diff
@@ -34,20 +34,23 @@ void getQuantity() {
         assertEquals(1,testStockType.getQuantity());
     }
 
+    //@@author patwaririshab
```

Comment:
don't need to spam your author tag, just enclose your code with it 

@@author rishab
`code here`
@@author

Reply:
thanks, fixed